<robot name="exp_box">
  <link name="base">
    <inertial>
      <origin xyz="0.075 0.115 0.0025"/>
      <mass value="0.5" />
      <inertia  ixx="0.0" ixy="0.0"  ixz="0.0"  iyy="0.0"  iyz="0.0"  izz="0.0" />
    </inertial>
    <visual>
      <origin xyz="0.075 0.115 0.0025"/>
      <geometry>
        <box size="0.15 0.23 0.005" />
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.075 0.115 0.0025"/>
      <geometry>
        <box size="0.15 0.23 0.005" />
      </geometry>
    </collision>
  </link>

  <link name="side_b1">
    <inertial>
      <origin xyz="0.0025 0.115 0.0325"/>
      <mass value="0.5" />
      <inertia  ixx="0.0" ixy="0.0"  ixz="0.0"  iyy="0.0"  iyz="0.0"  izz="0.0" />
    </inertial>
    <visual>
      <origin xyz="0.0025 0.115 0.0325"/>
      <geometry>
        <box size="0.005 0.23 0.065"/>
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.0025 0.115 0.0325"/>
      <geometry>
        <box size="0.005 0.23 0.065"/>
      </geometry>
    </collision>
  </link>

 <link name="side_b2">
    <inertial>
      <origin xyz="0.0025 0.115 0.0325"/>
      <mass value="0.5" />
      <inertia  ixx="0.0" ixy="0.0"  ixz="0.0"  iyy="0.0"  iyz="0.0"  izz="0.0" />
    </inertial>
    <visual>
      <origin xyz="0.0025 0.115 0.0325"/>
      <geometry>
        <box size="0.005 0.23 0.065"/>
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.0025 0.115 0.0325"/>
      <geometry>
        <box size="0.005 0.23 0.065"/>
      </geometry>
    </collision>
  </link>

 <link name="side_s1">
    <inertial>
      <origin xyz="0.08 0.0025 0.0325"/>
      <mass value="0.5" />
      <inertia  ixx="0.0" ixy="0.0"  ixz="0.0"  iyy="0.0"  iyz="0.0"  izz="0.0" />
    </inertial>
    <visual>
      <origin xyz="0.08 0.0025 0.0325"/>
      <geometry>
        <box size="0.16 0.005 0.065"/>
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.08 0.0025 0.0325"/>
      <geometry>
        <box size="0.16 0.005 0.065"/>
      </geometry>
    </collision>
  </link>

 <link name="side_s2">
    <inertial>
      <origin xyz="0.08 0.0025 0.0325"/>
      <mass value="0.5" />
      <inertia  ixx="0.0" ixy="0.0"  ixz="0.0"  iyy="0.0"  iyz="0.0"  izz="0.0" />
    </inertial>
    <visual>
      <origin xyz="0.08 0.0025 0.0325"/>
      <geometry>
        <box size="0.16 0.005 0.065"/>
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.08 0.0025 0.0325"/>
      <geometry>
        <box size="0.16 0.005 0.065"/>
      </geometry>
    </collision>
  </link>

 <link name="world"/>
 <joint name="box_fixed" type="fixed">
    <parent link="world"/>
	<child link="base"/>
 </joint>

 <joint name="base_to_b1" type="fixed">
    <parent link="base"/>
    <child link="side_b1"/>
    <origin xyz="-0.005 0.0 0.0"/>
  </joint>

 <joint name="base_to_b2" type="fixed">
    <parent link="base"/>
    <child link="side_b2"/>
    <origin xyz="0.15 0.0 0.0"/>
  </joint>

 <joint name="base_to_s1" type="fixed">
    <parent link="base"/>
    <child link="side_s1"/>
    <origin xyz="-0.005 -0.005 0.0"/>
  </joint>

  <joint name="base_to_s2" type="fixed">
    <parent link="base"/>
    <child link="side_s2"/>
    <origin xyz="-0.005 0.23 0.0"/>
  </joint>

 
  <gazebo reference="base">
    <material>Gazebo/Purple</material>
     <mu1>1000</mu1>
     <mu2>1000</mu2>
  </gazebo>

  <gazebo reference="side_b1">
    <material>Gazebo/Purple</material>
     <mu1>1000</mu1>
     <mu2>1000</mu2>
  </gazebo>

  <gazebo reference="side_b2">
    <material>Gazebo/Purple</material>
     <mu1>1000</mu1>
     <mu2>1000</mu2>
  </gazebo>

  <gazebo reference="side_s1">
    <material>Gazebo/Purple</material>
     <mu1>1000</mu1>
     <mu2>1000</mu2>
  </gazebo>

  <gazebo reference="side_s2">
    <material>Gazebo/Purple</material>
     <mu1>1000</mu1>
     <mu2>1000</mu2>
  </gazebo>
</robot>
